source/js/kitchen.js

```diff
@@ -146,24 +162,35 @@ function loadTemplateFromURL(templateURL, callback) {
     });
 }
 
+function presenterForType(type) {
+    console.log(type);
```

Comment:
It'll run on Xcode but if you hit the develop menu in Safari you'll see the iOS Simulator and will be able to see logs in the console.